portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/controlpanel/apps/appmanager/AppManagement.testcase

```diff
@@ -552,6 +552,46 @@ definition {
 			appStatus = "Active");
 	}
 
+	@description = "This test asserts that user can filter app by Price in marketplace store."
+	@priority = "3"
+	test CanFilterMarketplaceStoreAppsByPrice {
+		property test.name.skip.portal.instance = "AppManagement#CanFilterMarketplaceStoreAppsByPrice";
+
+		task ("Given user login marketplace") {
+			ApplicationsMenu.gotoPortlet(
+				category = "Marketplace",
+				panel = "Control Panel",
+				portlet = "Store");
+
+			AppManager.loginMarketplaceStore();
+
+			Navigator.openURL();
+		}
+
+		task ("When user filter by Price with value Free") {
+			ApplicationsMenu.gotoPortlet(
```

Comment:
Could you please explain again why we need navigate to the Store again? Sorry and thanks.
If it is trying to avoid some issue, we can add a description for it.

Reply:
Because when we first navigate the filter page can't filter free price successfully. For this unstable issue I find navigating the page 2 times it works. I will add the description for it.

Reply:
Done